Dockerfile

```diff
@@ -74,3 +75,37 @@ RUN	ln -s /opt/local/lib/livepeer-w3/livepeer-w3.js /usr/local/bin/livepeer-w3 &
 EXPOSE	1935	4242	8080	8889/udp
 
 CMD	["/usr/local/bin/MistController", "-c", "/etc/livepeer/catalyst.json"]
+
+FROM catalyst AS livepeer-in-a-box
+
+RUN	apt update && apt install -yqq \
+	rabbitmq-server \
+	nginx \
+	gdb \
+	inotify-tools \
+	file \
+	# for `shasum`
+	perl \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN curl -L -O https://binaries.cockroachdb.com/cockroach-v23.1.5.linux-amd64.tgz \
```

Comment:
There's an argument to be made that this could all be done through the manifest and `catalyst-downloader` but that'd involve splitting out the manifest into "catalyst stuff" and "box stuff" which let's hold off for now.